specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-03-01-preview/subscriptions.json

```diff
@@ -84,6 +84,39 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/enable": {
+      "post": {
+        "operationId": "Subscriptions_Enable",
```

Comment:
It is recommended to use camelCase when naming operationId

Reply:
The current naming convention is used in all of our other APIs.